src/config/dark-sites.config

```diff
@@ -51,6 +51,7 @@ ara-ara-ufufu.herokuapp.com
 arche-dev.codeberg.page
 archive.ragtag.moe
 argoto.snaz.in
+ari-web.xyz
```

Comment:
You can remove the other entries and only leave this one in.

Reply:
@Gusted Ah thanks, wait

Reply:
done